app/src/main/java/org/schabi/newpipe/player/Player.java

```diff
@@ -401,6 +401,17 @@ public Player(@NonNull final MainPlayer service) {
         audioResolver = new AudioPlaybackResolver(context, dataSource);
 
         windowManager = ContextCompat.getSystemService(context, WindowManager.class);
+
+        playbackStateListener = new PlaybackStateListener();
+        playerPlaybackListener = new PlayerPlaybackListener();
+        playerImageLoaderListener = new PlayerImageLoadingListener();
+        playerVideoListener = new PlayerVideoListener();
+
+        playerOnSeekBarChangeListener = new PlayerOnSeekBarChangeListener();
+        playerViewOnClickListener = new PlayerViewOnClickListener();
+        playerViewOnLongClickListener = new PlayerViewOnLongClickListener();
+        playerPopupMenuOnMenuItemClickListener = new PlayerPopupMenuOnMenuItemClickListener();
+        playerPopupMenuOnDismissListener = new PlayerPopupMenuOnDismissListener();
```

Comment:
Just initialize them directly in the `class` body.

Reply:
will change that